### PR TITLE
Auth api integration

### PR DESCRIPTION
**Pull request by WJKw0k**

### CODE REVIEW

File: lexic-backend/backend.js [hunk 1]
```diff
@@ -33,10 +33,20 @@ app.get("/", (req, res) => {
   res.send("Hello World!");
 });
 
-app.get("/guess/:word", async (req, res) => {
+// AUTHENTICATION HERE
+app.get("/guess/:word/:name", async (req, res) => {
+  console.log('initial req body', req.body);
+  console.log('initial headers', req.headers);
+  console.log('params', req.params);
+  const userInfo = await authAndCheckExistence(req);
```
Comment (Kanaz312) [inline lexic-backend/backend.js:41]: Needs branch if auth failed

**Review by Kanaz312 — CHANGES_REQUESTED** [review]

**Review by Kanaz312 — APPROVED** [review]
Looks good to me!

**Review by dbolusan — APPROVED** [review]
Looks Good to Me

**Review by Tylertyca — APPROVED** [review]
Looks good

**Pull request merged**